core/test/usLDAPQueryTest.cpp

```diff
@@ -0,0 +1,123 @@
+/*=============================================================================
+ 
+ Library: CppMicroServices
+ 
+ Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+ file at the top-level directory of this distribution and at
+ https://github.com/saschazelzer/CppMicroServices/COPYRIGHT .
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ 
+ =============================================================================*/
+
+#include <usFrameworkFactory.h>
+#include <usFramework.h>
+
+#include <usBundle.h>
+#include <usLDAPFilter.h>
+#include <usLog.h>
+
+#include "usTestUtils.h"
+#include "usTestingMacros.h"
+
+using namespace us;
+
+void TestLDAPQueryBundleMetaData(BundleContext* frameworkCtx)
```

Comment:
If usTestBundleLQ_2 through usTestBundleLQ_5 do not export services and only contain meta-data, you shouldn't need to create dummy .cpp files for them. You should be able to generate "meta-data only" bundles. This will cut down on the number of files to maintain.